src/database/schema/ChatSchema.ts

```diff
@@ -270,7 +274,7 @@ class ChatSchema implements BaseDbSchema {
       console.log(e);
     }
     try {
-      if (json?.message?.message_type !== 'reply_prompt') {
+      if (isSavingAttachmentAllowed(json?.message?.message_type)) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for saving attachments has been refactored into a separate function `isSavingAttachmentAllowed` which checks if the message type allows saving attachments. This improves code readability and maintainability.
```diff
- if (json?.message?.message_type !== 'reply_prompt') {
+ if (isSavingAttachmentAllowed(json?.message?.message_type)) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->